velox/dwio/parquet/writer/Writer.h

```diff
@@ -35,12 +35,16 @@ class ArrowDataBufferSink;
 
 struct ArrowContext;
 
+constexpr int64_t kDefaultBytesInRowGroup = 128 * 1'024 * 1'024;
```

Comment:
Would you add comments for these values and move them into the DefaultFlushPolicy class?

Use 128 << 20 to match the rest of the code base.